hermes.js

```diff
@@ -311,9 +309,7 @@ function runTest(parents, test, sublevel) {
     }
 
     if (test.subtests) {
-        var newParents = parents.slice(0);
-        newParents.push(test.name);
-        test.subtests.forEach(function (v) { runTest(newParents, v, sublevel + 1); });
+        test.subtests.forEach(function (v) { runTest(parents.concat([test.name]), v, sublevel + 1); });
```

Comment:
```suggestion
        test.subtests.forEach(function (v) { runTest(parents.concat(test.name), v, sublevel + 1); });
```
unless test.name can be an array